src/tests/ScoreBoard.spec.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from 'react-testing-library';
+import { ScoreBoard } from '../ScoreBoard';
+
+describe('ScoreBoard component', () => {
```

Comment:
add a test for checking that the order of scores is correct. The order should be, first the user with minimum moves and at last the user with most moves